packages/mainmenu-extension/src/index.ts

```diff
@@ -369,13 +369,13 @@ function createFileMenu(
 
   commands.addCommand(CommandIDs.shutdown, {
     label: trans.__('Shut Down'),
-    caption: trans.__('Shut down JupyterLab'),
+    caption: trans.__('Shut down "%1"', app.name),
```

Comment:
I think you took the examples a bit too literally. 

```
caption: trans.__('Shut down "%1"', app.name),
```

Will render as `Shut down "JupyterLab"` / `Shut down "Notebook"`; only the `%1` will be replaced. I think we want without the quotes in the various places. But a core dev can confirm.

Otherwise this seem good to me,

Reply:
I agree with the above.

Reply:
```suggestion
    caption: trans.__('Shut down %1', app.name),
```